addon/data/set-installed-flag.js

```diff
@@ -10,3 +10,8 @@
 
 unsafeWindow.navigator.testpilotAddon = true;
 unsafeWindow.navigator.testpilotAddonVersion = self.options.version;
+
+self.port.on('detach', function() {
+  unsafeWindow.navigator.testpilotAddon = false;
+  unsafeWindow.navigator.testpilotAddonVersion = null;
```

Comment:
can we delete these instead? It doesn't seem to matter in this case I suppose, except that by deleting them its a clearer signal (to me) that they aren't used by anything else anymore.